src/cmdexpand.c

```diff
@@ -3019,6 +3026,12 @@ ExpandFromContext(
 	return ExpandRTDir(pat, DIP_START + DIP_OPT, numMatches, matches,
 								directories);
     }
+    if (xp->xp_context == EXPAND_RUNTIME)
+    {
+	char *directories[] = {"", NULL};
+	return ExpandRTDir(pat, DIP_START + DIP_OPT + DIP_PRNEXT, numMatches, matches,
+								directories);
```

Comment:
From the above codecov output, it looks like most of the key changes are not covered by tests. Can you modify the tests to cover these lines?